src/pages/ThorChainLP/YourPositions.tsx

```diff
@@ -182,87 +66,209 @@ export const YourPositions = () => {
     return allUserLpData.filter(query => query.data?.positions.length)
   }, [allUserLpData])
 
-  // If some are loading, we are loading, but that's not it yet, we also need to check if some are not loaded
-  const someLoading = useMemo(
-    () => allUserLpData.some(position => position.isLoading),
-    [allUserLpData],
-  )
-
   const allLoaded = useMemo(() => {
     return allUserLpData.length && allUserLpData.every(query => query.isSuccess)
   }, [allUserLpData])
 
-  const isEmpty = useMemo(() => allLoaded && !activePositions.length, [allLoaded, activePositions])
-
   const emptyIcon = useMemo(() => <PoolsIcon />, [])
   const headerComponent = useMemo(() => <PoolsHeader />, [])
 
-  const positionRows = useMemo(() => {
-    if (isEmpty) {
-      return (
-        <ResultsEmpty
-          title='pools.yourPositions.emptyTitle'
-          body='pools.yourPositions.emptyBody'
-          icon={emptyIcon}
-        />
-      )
-    }
+  const positions = useMemo(
+    () =>
+      activePositions
+        .flatMap(position => position?.data?.positions ?? [])
+        .filter(isSome)
+        .sort((a, b) => bn(b.totalValueFiatUserCurrency).comparedTo(a.totalValueFiatUserCurrency)),
+    [activePositions],
+  )
 
-    const skeletons = new Array(2).fill(null).map((_, i) => <Skeleton height={16} key={i} />)
+  const columns: Column<UserLpDataPosition>[] = useMemo(
+    () => [
+      {
+        Header: translate('pools.pool'),
+        accessor: 'name',
+        Cell: ({ row }: { value: string; row: RowProps }) => {
+          const position = row.original
+          const { assetId } = useMemo(() => position, [position])
+          const poolAssetIds = useMemo(() => [assetId, thorchainAssetId], [assetId])
 
-    const positions = activePositions
-      .flatMap(position => position?.data?.positions ?? [])
-      .filter(isSome)
-      .sort((a, b) => bn(b.totalValueFiatUserCurrency).comparedTo(a.totalValueFiatUserCurrency))
+          return (
+            <Flex gap={4} alignItems='center'>
+              <PoolIcon assetIds={poolAssetIds} size='sm' />
+              <RawText>{position.name}</RawText>
+              <Tag>
+                {position.asym ? (
+                  <Text
+                    // eslint-disable-next-line react-memo/require-usememo
+                    translation={['common.asymmetric', { assetSymbol: position.asym.asset.symbol }]}
+                  />
+                ) : (
+                  <Text translation='common.symmetric' />
+                )}
+              </Tag>
+            </Flex>
+          )
+        },
+      },
+      {
+        Header: translate('pools.balance'),
+        accessor: 'totalValueFiatUserCurrency',
+        Cell: ({ value, row }: { value: string; row: RowProps }) => {
+          const position = row.original
+          const { assetId } = useMemo(() => position, [position])
```

Comment:
just wondering what memoizing this does?

Reply:
Definitely fluff, nice catch! 2a87ec23c93d079f003a12c23973a921de421449